contracts/optimistic-ethereum/OVM/verification/OVM_StateTransitioner.sol

```diff
@@ -70,13 +72,15 @@ contract OVM_StateTransitioner is iOVM_StateTransitioner, Lib_AddressResolver {
     )
         Lib_AddressResolver(_libAddressManager)
     {
+        // TODO: Can this be removed?
         stateTransitionIndex = _stateTransitionIndex;
         preStateRoot = _preStateRoot;
         postStateRoot = _preStateRoot;
         transactionHash = _transactionHash;
 
         ovmExecutionManager = iOVM_ExecutionManager(resolve("OVM_ExecutionManager"));
         ovmStateManager = iOVM_StateManagerFactory(resolve("OVM_StateManagerFactory")).create(address(this));
+        ovmBondManager = OVM_BondManager(resolve("OVM_BondManager"));
```

Comment:
Hmmm now that I'm thinking about it, do we also want to include the users who call `initializeFraudVerification` and `finalizeFraudVerification` in the reward? @ben-chain @karlfloersch

Reply:
https://github.com/ethereum-optimism/contracts-v2/pull/3/commits/932582a2f41baee64ed338bb1e3c2c4ac443bcfb done, check the commit message for adjusting your deploy scripts

Reply:
This seems reasonable!